classes/helper/login_helper.php

```diff
@@ -518,17 +557,50 @@ public function is_username_set_in_auth_global_vars() {
      * Verifies that the user is already logged in.
      *
      * @return bool
+     * @throws \coding_exception
      */
-    public function is_user_already_loggedin() {
+    public function is_user_already_loggedin()
+    {
         return isloggedin() and !isguestuser();
     }
 
     /**
      * Clear the username from
      */
-    public function clear_username_in_auth_global_vars() {
+    public function clear_username_in_auth_global_vars()
+    {
         global $frm;
         $frm = null;
         set_moodle_cookie('');
     }
+
+    /**
+     * Get user failed login count
+     *
+     * @param $userName
+     * @return int|null
+     * @throws \coding_exception
+     * @throws \dml_exception
+     */
+    public function get_user_login_failCount($userName)
```

Comment:
Can we be consistent with naming here please - either underscores as per the rest of file, or camelCase, but not both in the same function name

Reply:
Done, please review